frontend/public/style/_overrides.scss

```diff
@@ -203,3 +203,7 @@ tags-input .autocomplete .suggestion-item em {
     border-bottom: 1px solid $table-border-color;
   }
 }
+
+.btn.btn-link {
+  font-size: $font-size-base;
```

Comment:
I'm worried this will cause problems in other contexts. I'd rather use this only where we need it.

Reply:
Meant to remove this.